package.json

```diff
@@ -46,7 +47,7 @@
     "npm-run-all": "4.1.5",
     "prettier": "^2.0.5",
     "ts-node": "^8.10.2",
-    "typescript": "^3.9.6"
+    "typescript": "^4.3.2"
```

Comment:
any break changes between TS v3.9.x and v4.3.x?

Reply:
Let me edit it to v3.9.9.